engine/lobby.gd

```diff
@@ -103,6 +103,14 @@ func _ready():
 	get_tree().connect("server_disconnected",self,"_server_disconnected")
 	
 	get_tree().set_auto_accept_quit(false)
+
+	if OS.get_name() == "Server":
```

Comment:
As a note here, I believe that @Nodding is working on different ways to pass the values to configure the server.

Out of this section the only essential portion is:
```
if OS.get_name() == "Server":
  call_deferred("_on_host_pressed", port)
```
because this causes the server to start automatically if running from the Godot server executable (which is used in the Docker image)